kanidm_unix_int/pam_kanidm/src/pam/mod.rs

```diff
@@ -30,3 +30,332 @@ pub mod items;
 #[doc(hidden)]
 pub mod macros;
 pub mod module;
+
+#[cfg(target_family = "unix")]
```

Comment:
Since this is the module, and we have the cfg(unix) above in lib.rs, do we need all these cfgs?

Reply:
Nope, still need to clean it up (done)